ux.symfony.com/importmap.php

```diff
@@ -184,7 +185,28 @@
     'scheduler' => [
         'version' => '0.23.0',
     ],
-    '@symfony/ux-live-component' => [
-        'path' => './vendor/symfony/ux-live-component/assets/dist/live_controller.js',
+    'tippy.js' => [
+        'version' => '6.3.7',
+    ],
+    'tippy.js/dist/tippy.css' => [
+        'version' => '6.3.7',
+        'type' => 'css',
+    ],
+    'tippy.js/themes/translucent.css' => [
+        'version' => '6.3.7',
+        'type' => 'css',
+    ],
+    'path-to-regexp' => [
+        'version' => '6.2.1',
+    ],
+    '@swup/theme' => [
+        'version' => '2.1.0',
+    ],
+    '@kurkle/color' => [
+        'version' => '0.3.2',
+    ],
+    'bootstrap/dist/css/bootstrap.min.css' => [
+        'version' => '5.3.3',
+        'type' => 'css',
```

Comment:
We could remove this to avoid confusion, as we're not using this directly.

Reply:
Did i add this ? oO